src/app/product-detail/product-detail.component.html

```diff
@@ -0,0 +1,76 @@
+<div class="wrapper">
+  <div class="products">
+    <button class="go-back" [text]="true" pButton>
+      <i class="pi pi-arrow-left"><b>Products</b></i>
+    </button>
```

Comment:
remove <i> tag and use the icon="pi pi-check" as button input.
check docs [text](https://primeng.org/button#icons)

Reply:
remove i tag*